src/toil/wdl/wdl_functions.py

```diff
@@ -348,6 +348,34 @@ def process_and_read_file(f, tempDir, fileStore, docker=False):
     return read_file(processed_file, tempDir, fileStore, docker=docker)
 
 
+def generate_stdout_file(stdout, tempDir, stderr=False, fileStore=None):
+    """
+    Create a stdout (or stderr) file from a string or bytes object.
+    Returns a file path.
+
+    :param stdout: str|bytes
+    :param tempDir: str directory
+    :param stderr: bool
```

Comment:
I think these are supposed to be `:param type varname: description of meaning`, and not `:param varname: type`.